wpilib/wpilib/smartdashboard.py

```diff
@@ -77,7 +85,7 @@ def putData(cls, *args, **kwargs):
         - value
         
         :param key: the key (cannot be None)
-        :type  key: str
+        :type key: str
```

Comment:
Not too sure why you changed this line, but ok.

Reply:
I'm surprised black didn't take care of it anyway

Reply:
black doesn't format docstrings. There are too many different docstring conventions to make that possible for a one-size-fits-all tool.